ocaml_version.ml

```diff
@@ -364,6 +371,10 @@ module Configure_options = struct
       | `Frame_pointer -> "--enable-frame-pointers"
       | `No_naked_pointers -> "--disable-naked-pointers"
       | `Disable_flat_float_array -> "--disable-flat-float-array"
+      | `Mingw `X86_64 ->
+         "--host=x86_64-w64-mingw32 CC=x86_64-w64-mingw32-gcc CPP=x86_64-w64-mingw32-cpp"
```

Comment:
`CC` and `CPP` should not need overriding?

Reply:
I wasn't sure, I saw the build script in opam-repository-mingw did override them, so I reluctantly added the two vars.

Reply:
Yeah, that override drives me up the wall developing (I've been stung by it several times on machines which happen to have it installed!). It definitely won't make it upstream...